docs/reference/quick-start.asciidoc

```diff
@@ -0,0 +1,284 @@
+
+To keep your data organized, each document in an index requires a unique ID. You
+can provide this ID when you index a document or let {es} generate one for
+you.
+[float]
+==== Provide a document ID
+
+To index a document with a specific ID, use the PUT method with the
+<<docs-index_, index API>>.
+
+This example command adds a document with an ID of `1` to the `customer` index.
+This command will also create the `customer` index if it doesn't already exist.
+
+[source, js]
+--------------------------------------------------
+PUT /customer/_doc/1
+{
+    "firstname": "Amber",
+    "lastname": "Duke",
+    "age": 32,
+    "employer": "Pyrami"
+}
+--------------------------------------------------
+// CONSOLE
+// TESTSETUP
+
+You can confirm {es} used your ID by checking the `\_id` field of the response.
+Here's an example response, formatted to be more readable.
+
+[source, js]
+--------------------------------------------------
+{
+  "_index" : "customer",
+  "_type" : "_doc",
+  "_id" : "1",
+  "_version" : 2,
+  "result" : "updated",
+  "_shards" : {
+    "total" : 2,
+    "successful" : 1,
+    "failed" : 0
+  },
+  "_seq_no" : 1,
+  "_primary_term" : 1
+}
+--------------------------------------------------
+// TESTRESPONSE
+
+[float]
+==== Autogenerate a document ID
+
+To add a document without providing an ID, use the POST method with the
+<<docs-index_, index API>>. {es} will automatically generate an ID for the
+document.
+
+This example command adds a document to `customer` index. This command will
+also create the `customer` index if it doesn't already exist.
+
+[source, js]
+--------------------------------------------------
+POST /customer/_doc
+{
+    "firstname": "Dale",
+    "lastname": "Adams",
+    "age": 48,
+    "employer": "Geekfarm"
+}
+--------------------------------------------------
+// CONSOLE
+
+{es} will provide the generated ID in `\_id` field of the response. Here's an
+example response, formatted to be more readable.
+
+[source, js]
+--------------------------------------------------
+{
+  "_index" : "customer",
+  "_type" : "_doc",
+  "_id" : "yDWh7WkBJ8uiWiWqEEaF",
+  "_version" : 2,
+  "result" : "updated",
+  "_shards" : {
+    "total" : 2,
+    "successful" : 1,
+    "failed" : 0
+  },
+  "_seq_no" : 1,
+  "_primary_term" : 1
+}
+--------------------------------------------------
+// TESTRESPONSE[s/yDWh7WkBJ8uiWiWqEEaF/$body._id/]
+
+[float]
+=== Run a search
+
+Now that you have some documents stored in Elasticsearch, you can fetch
+and search your data.
+
+[float]
+==== Get a document by ID
+
+To get a document based on its ID, use the <<docs-get, get API>>.
+
+This example returns the document with an ID of `1` in the
+`customer` index.
+
+[source, js]
+--------------------------------------------------
+GET /customer/_doc/1
+--------------------------------------------------
+// CONSOLE
+
+[float]
+==== URI search
+For simpler test searches, you can use {es}'s <<search-search, search API>>
```

Comment:
I try to avoid modifiers on attributes, in case that causes translation problems later. Here's a suggestion:

```suggestion
For simpler test searches, you can use the {es} <<search-search, search API>>
```